builtin/logical/transit/path_import_test.go

```diff
@@ -600,7 +601,7 @@ func generateKey(keyType string) (interface{}, error) {
 	switch keyType {
 	case "aes128-gcm96":
 		return uuid.GenerateRandomBytes(16)
-	case "aes256-gcm96":
+	case "aes256-gcm96", "hmac":
```

Comment:
I think we want to actually generate 64 bytes for hmac?

My 2c., but since SHA-256 uses a 512-bit block (which I think is what HMAC is using under the covers, but it isn't clear from my quick glance) -- a 32-byte key would be padded with 32 bytes of zeros, I'd rather we use all the bits since we can.

Reply:
We could add hash function as a sub param.  I think I like that rather than a plethora of distinct types.